cli/tests/integration/run_tests.rs

```diff
@@ -3642,6 +3649,13 @@ itest!(no_lock_flag {
   exit_code: 0,
 });
 
+itest!(config_file_lock_false {
+  args: "run run/config_file_lock_false/main.ts",
```

Comment:
Does this test fail when you change `"lock": false` to `"lock": true`?

Reply:
Yes, it does.